lib/full_text_search/searcher.rb

```diff
@@ -242,10 +243,16 @@ def sort_keys
         direction = ""
       end
       case @request.order_target
-      when "date"
+      # TODO: Supporting the `date` parameter for backward compatibility,
+      # but it will be deprecated in the future. Use `last_modified_time` instead.
```

Comment:
We can make it deprecated right now.

Reply:
fix: [61f2096](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/61f20967cf7d53513c6f1314c37eba69cdf6135d)